xtas/core.py

```diff
@@ -84,7 +84,7 @@ def configure(config, import_error="raise", unknown_key="raise"):
 
     app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
     es = config.get('ELASTICSEARCH', _defaultconfig.ELASTICSEARCH)
-    _config['ELASTICSEARCH'] = es
+    config['ELASTICSEARCH'] = es
```

Comment:
I'm not for renaming a user-facing parameter because it looks too much like a private variable. We could rename `_config` to `_global_config` instead?

By the way, changing a parameter name is an API change in Python because every parameter can be used as a keyword arg:

```py
def foo(x): print(x)
foo(x=1)
```